clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -2881,6 +2885,31 @@ private static void dumpIngestionState(CommandLine cmd) throws Exception {
     }
   }
 
+  private static void addChangeCaptureView(CommandLine cmd) {
+    addChangeCaptureView(cmd, controllerClient);
+  }
+
+  protected static void addChangeCaptureView(CommandLine cmd, ControllerClient client) {
+    String storeName = getRequiredArgument(cmd, Arg.STORE);
+
+    // Because of the way validation is done in the controller, we need to do this in two steps
+    UpdateStoreQueryParams params = new UpdateStoreQueryParams().setActiveActiveReplicationEnabled(true)
+        .setChunkingEnabled(true)
+        .setRmdChunkingEnabled(true);
+    ControllerResponse response = client.updateStore(storeName, params);
+    if (response.isError()) {
+      printErrAndExit(response.getError());
+    }
+    Map<String, String> views = new HashMap<>();
+
+    views.put(
+        getOptionalArgument(cmd, Arg.VIEW_NAME, ChangeCaptureView.DEFAULT_CHANGE_CAPTURE_VIEW_NAME),
+        "{\"viewClassName\" : \"" + ChangeCaptureView.class.getCanonicalName() + "\", \"viewParameters\" : {}}");
```

Comment:
Should we support passing view params, or not necessary for this view...?

Reply:
I think we could/should add that if we can puzzle out better why passing json args on the command line isn't working as intended.  I didn't add it here because I was in a hurry.  Best option would be to figure out why json passing isn't working (and thereby, fix other commands which are currently broken) and then add that here as well.